### PR TITLE
Release 2.8.1

### PR DESCRIPTION
**Pull request by Log234**

### CODE REVIEW

File: src/common/LaviniaSVGLogo/LaviniaSVGLogo.tsx [hunk 1]
```diff
@@ -1,46 +1,50 @@
...
+                <path d="m71.177 91.703 0.93018 21.911h-0.49093v0.46509h-2.2479v-0.43925h-0.43925l0.93018-21.937z" />
+                <path d="m76.754 91.755 0.93018 21.911h-0.49093v0.46509h-2.2479v-0.43925h-0.43925l0.93018-21.937z" />
+                <path d="m79.277 119.42-45.891 8e-3s-1.1693-0.4933 0.38368-3.2521l45.124-8e-3 0.26458-1.9e-4c1.8176 2.7586 0.38368 3.2523 0.38368 3.2523z" />
+                <path d="m34.226 83.079-0.54811-0.54811 0.59379-0.59379h44.45l0.54811 0.54811-0.59378 0.59379z" />
+              </g>
+            </g>
+          </g>
+        </svg>
+      </div>
+    )
```
Comment (Log234) [inline src/common/LaviniaSVGLogo/LaviniaSVGLogo.tsx:48]: Codacy found an issue: [Missing semicolon.](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5948151)

**Comment by Log234** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5948151&bid=19498144)

File: src/Layout/Navigation/LaviniaSVGLogo.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  <path d="m71.177 91.703 0.93018 21.911h-0.49093v0.46509h-2.2479v-0.43925h-0.43925l0.93018-21.937z"/>
+  <path d="m76.754 91.755 0.93018 21.911h-0.49093v0.46509h-2.2479v-0.43925h-0.43925l0.93018-21.937z"/>
+  <path d="m79.277 119.42-45.891 8e-3s-1.1693-0.4933 0.38368-3.2521l45.124-8e-3 0.26458-1.9e-4c1.8176 2.7586 0.38368 3.2523 0.38368 3.2523z"/>
+  <path d="m34.226 83.079-0.54811-0.54811 0.59379-0.59379h44.45l0.54811 0.54811-0.59378 0.59379z"/>
+  </g>
+  </g>
+  </g>
+  </svg>
+  </div>  
+    )
```
Comment (Log234) [inline src/Layout/Navigation/LaviniaSVGLogo.tsx:51]: Codacy found an issue: [Missing semicolon.](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5948151)

File: src/requested-data/requested-data-reducer.ts [hunk 3]
```diff
@@ -51,3 +51,12 @@ export function requestedData(
             return state;
     }
 }
+
+function checkStateLoaded(state: RequestedDataState, action: RequestedDataAction): boolean {
```
Comment (Log234) [inline src/requested-data/requested-data-reducer.ts:55]: Codacy found an issue: [All 'checkStateLoaded' signatures should be adjacent](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5948151)

Comment (Log234) [inline src/requested-data/requested-data-reducer.ts:55]: Codacy found an issue: ['checkStateLoaded' is already defined.](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5948151)

**Comment by simendag** [comment]
I think something went wrong here. I must have messed up the pull from Development or something. I'll close this for a bit and see if I can fix it.

**Pull request closed without merging**